<!DOCTYPE html>
<head>
    <meta charset="utf-8">
    <title>Remdesivier</title>
    <link href="Cure Decor.css" rel="stylesheet">
</head>
<header>
    <div class="navi">
      <p><a href="index.html">Scan to Cure</a></p>
      <ul>
        <li><a href="#">Covid Overview</a>
          <ul>
            <li><a href="Structure.html">Genomic Characteritics of Corona Virus</a></li>
          </ul>
        </li>
          
        <li><a href="#">General Info of COVID Pro-Drugs</a>
          <ul>
            <li><a href="Remdesivier Abstract.html">Remdesivir</a></li>
            <li><a href="Fluvoxamine Abstract.html">Fluvoxamine</a></li>
            <li><a href="Molnupiravir Abstract.html">Molnupiravir</a></li>
          </ul>
        </li>
        <li><a href="#">Mechanism of Action</a>
          <ul>
            <li><a href='Remdesivier.html'>Remdesivir</a></li>
            <li><a href='Fluvoxamine.html'>Fluvoxamine</a></li>
            <li><a href='Molnupiravir.html'>Molnupiravir</a></li>
          </ul>
        </li>
        <li><a href='Similarities & Differences.html'>Similarities & Differences</a></li>
            <li><a href='Literature Sources.html'>Literature Source</a></li>
      </ul>
    </div>


  </header>

<body>
    <div class='for'>
        <h2>Remdesivir</h2>
        <h4>Abstract</h4>
        <br>
        <p>
            Remdesivir has broad-spectrum antiviral action against many single-stranded RNA  viruses including pathogenic SAR-CoV and the Middle East respiratory syndrome  coronavirus (MERS-CoV). In this study, we proposed that remdesivir strongly binds to  membrane protein (Mprotein), RNA-dependent RNA polymerase (RDRP), and main  protease (Mprotease) of SARS- CO V-2. The structure dynamics study suggested binding of  remdesivir leads to unfolding of RDRP. Binding of remdesivir to Mprotease, Mprotein, and  RDRP reduces the average motions in protein due to its strong binding. Remdesivir is a  metabolically active from (GS-441524) that works by inhibiting viral RNA-dependent RNA  polymerase (RDRP) while evading proofreading by viral exoribonuclease, which leads to  premature termination of viral RNA transcription and hence inhibits the spread of the virus  as well as the production of viral RNA (Wang M. et al., 2020). It showed possible binding  and inhibitory mechanisms of remdesivir on the C-terminal domain of SARS-CoV-2  nucleocapsid protein (CTD), envelope protein (Eprotein), main protease (Mprotease),  membranes protein (Mprotease), membrane phosphoprotein (NTD), RDRP, and spike  protein (Sprotein) using several computational approaches. It shows strong inhibitory effects  on Mprotein, RDRP, and spike protein (Sprotein, Figure 1G) using several computational  approaches. It shows strong inhibitory effects on Mprotein, RDRP (Figure 1D), and  Mprotease of SARS-CoV-2.  
        </p>
        <hr>
        <h4>Modeling</h4>
        <br>
        <p>
            The crystal structures of CTD nucleocapsid (PDB: 7c22), Mprotease (PDB: 7k40),  NTD nucleocapsid (PDB: 7acs), RDRP (RDB: 6m71), and Sprotein (PDB: 6vsb) were  obtained from the PDB (https://www.rcsb.org/), and the structure of remdesirvir was  obtained from the PubChem (CID 121304016).  
        </p>
        <hr>

        <h4>Discussion</h4>
        <br>
        <p>The SARS-CoV-2 genome is made of 29,891 nucleotides with 9,860 amino acids  (Chan et al., 2020), packaged in a circular nucleocapsid protein and encapsulated by  envelope proteins (Li, 2016). All proteins included in this study, namely, CTD, Eprotein,  Mprotease, Mprotein, NTD, RDRP, and Sprotein, play significant roles and hence are  promising therapeutic targets to control COVID-19 (Satarker and Nampoothiri, 2020). Each  figure (1-8) shows how fast and how much of the area of each nucleocapsid(CTD) protein  that interacts with remdesivir. Respectively time is given in nanoseconds(ns) and the area in  nanometers(nm). Remdesivir binds with CTD, Eprotein, Mprotease, Mprotein, NTD, RDRP,  and Sprotein with –4.8, -4.2, -7.8, -7.4, -6.2, -7.1, and –5.8 kcal/mol, respectively (Figure 1).  It shows several electrostatic and van der Walls interactions as listed in Table 1. Remdesivir  shows weak van der Waals interactions with CTD and Eprotein. It shows strong binding  affinity with Mprotease, Mprotein, and RDRP. Each figure(1-8) showw fast and how much  of an area of each protein and protease that remdesivir can interact with.
        </p>
        <hr>
        <p><img src="https://www.frontiersin.org/files/Articles/710778/fphar-12-710778-HTML/image_m/fphar-12-710778-g001.jpg"></p>
        <p><strong>FIGURE 1.</strong> Molecular docking of remdesivir into the active pocket of CTD, Eprotein, Mprotease, Mprotein, NTD, RDRP, and Sprotein, respectively. The structure indicated different residual interactions in (A) CTD–remdesivir, (B) Eprotein–remdesivir, (C) Mprotease–remdesivir, (D) Mprotein–remdesivir, (E) NTD–remdesivir, (F) RDRP–remdesivir, and (G) Sprotein–remdesivir, respectively. The electrostatic, van der Waals, and covalent bonds are represented in pink, green, and purple colors.
        </p>
        <hr>
        <br>
        <p><img src="https://www.frontiersin.org/files/Articles/592908/fmicb-11-592908-HTML/image_m/fmicb-11-592908-t001.jpg" height="400" width="900"></p>
        <p><strong>TABLE 1.</strong> Molecular docking of remdesivir with CTD, Eprotein, Mprotease, Mprotein, NTD, RDRP, and Sprotein, respectively.
        </p>
        <hr>
        <h4>Structural Dynamics of Eprotein in Binding Mechanism</h4>
        <br>
        <p>The average RMSD(Remdesivir) values of Eprotein and Eprotein–remdesivir were  found to be 0.50 and 0.49 nm, respectively. Remdesivir was found to have random  fluctuations in the active pocket of Eprotein (Figure 3A). It has been found that average  RMSD values were almost the same upon binding of remdesivir. It has been found that  Eprotein is not inhibited or unfolded upon binding of remdesivir. In other words, remdesivir  did not show inhibitory effects on Eprotein.  
        </p>
        <hr>
        <br>
        <p><img src= "https://www.frontiersin.org/files/Articles/710778/fphar-12-710778-HTML/image_m/fphar-12-710778-g003.jpg"></p>
        <p><strong>FIGURE 3.</strong> Structural dynamics of Eprotein. (A) Root mean square deviation plot for  Eprotein (black), Eprotein–remdesivir (red), and remdesivir (green) as a function of time.
        </p>
        <hr>
        <h4>Structural Dynamics of Mprotease in Binding Mechanisms:</h4>
        <br>
        <p>Mprotease plays a key role in coordinating viral replication and transcription of the  virus life cycle. Therefore, Mprotease is a prime target of drugs for SARS-CoV-2 (Liu et al.,  2020; Naik et al., 2020). The average RMSD values of Mprotease and Mprotease–remdesivir  were found to be 0.32 and 0.33 nm, respectively. Remdesivir was found to have constant  fluctuations in the active pocket of Mprotease (Figure 4A). It has been found that average  RMSD values were the same upon binding of remdesivir. 
        </p>
        <hr>
        <br>
        <p><img src="https://www.frontiersin.org/files/Articles/710778/fphar-12-710778-HTML/image_m/fphar-12-710778-g004.jpg"></p>
        <p><strong>FIGURE 4.</strong> Structural dynamics of Mprotease. (A) Root mean square deviation plot for  Mprotease (black), Mprotease–remdesivir (red), and remdesivir (green) as a function of time.
        </p>
        <hr>
        <h4>Structural Dynamics of Mprotein in Binding Mechanisms:</h4>
        <br>
        <p>The average RMSD values of Mprotein and Mprotein–remdesivir were found to be  0.56 and 0.47 nm, respectively. Remdesivir showed constant fluctuations in the active pocket  of Mprotein (Figure 5A). It has been found that average RMSD values were less upon  binding of remdesivir. This indicates that remdesivir binds tightly with Mprotein and  possibly inhibits it.</p>
        <hr>
        <p><img src="https://www.frontiersin.org/files/Articles/710778/fphar-12-710778-HTML/image_m/fphar-12-710778-g005.jpg"></p>
        <p><strong>FIGURE 5</strong>. Structural dynamics of Mprotein. (A) Root mean square deviation plot for  Mprotein (black), Mprotein–remdesivir (red), and remdesivir (green) as a function of time</p>
        <hr>
        <h4>Structural Dynamics of NTD in Binding Mechanisms:</h4>
        <br>
        <p>The average RMSD values of NTD and NTD–remdesivir were found to be 0.50 and  0.64 nm, respectively. Additionally, remdesivir showed random fluctuations in the active  pocket of NTD (Figure 6A). It has been found that average RMSD values were more upon  binding of remdesivir. This indicates that the random fluctuations increased upon remdesivir  binding to NTD</p>
        <hr>
        <p><img src="https://www.frontiersin.org/files/Articles/710778/fphar-12-710778-HTML/image_m/fphar-12-710778-g006.jpg"></p>
        <p><strong>FIGURE 6.</strong> Structural dynamics of NTD. (A) Root mean square deviation plot for NTD  (black), NTD–remdesivir (red), and remdesivir (green) as a function of time.
        </p>
        <h4>Structural Dynamics of RNA-Dependent RNA Polymerase in Binding Mechanisms:</h4>
        <br>
        <p>It has been reported that remdesivir inhibits RDRP and shows antiviral activity  against multiple variants of the Ebola virus (Warren et al., 2016). It is also expected that  remdesivir inhibits RDRP of SARS-CoV-2. The average RMSD values of RDRP and  RDRP–remdesivir were found to be 0.68 and 0.67 nm, respectively. Additionally, remdesivir  was stable in the active pocket of RDRP (Figure 7A). It has been found that the structure of  RDRP was not deviated upon binding of remdesivir.  
        </p>
        <hr>
        <p><img src="https://www.frontiersin.org/files/Articles/710778/fphar-12-710778-HTML/image_m/fphar-12-710778-g007.jpg"></p>
        <p><strong>FIGURE 7.</strong>Structural dynamics of RDRP. (A) Root mean square deviation plot for RDRP  (black), RDRP–remdesivir (red), and remdesivir (green) as a function of time.</p>
        <hr>
        <h4>Structural Dynamics of Sprotein in Binding Mechanisms:</h4>
        <br>
        <p>The average RMSD values of Sprotein and Sprotein–remdesivir were found to be  1.90 and 2.29 nm, respectively. Remdesivir maintained stable conformation in the active  pocket of Sprotein from 25 to 100 ns MD simulations (Figure 8A). 
        </p>
        <hr>
        <p><img src="https://www.frontiersin.org/files/Articles/710778/fphar-12-710778-HTML/image_m/fphar-12-710778-g008.jpg"></p>
        <p><strong>FIGURE 8.</strong> Structural dynamics of Sprotein. (A) Root mean square deviation plot for  Sprotein (black), Sprotein–remdesivir (red), and remdesivir (green) as a function of time.
        </p>
        <hr>
        <h4>Hydrogen Bonding:</h4>
        <br>
        <p>The hydrogen bond is a significant factor in stabilizing protein conformations. To  check the hydrogen bond formations between remdesivir and SARS-CoV-2 proteins such as  CTD, Eprotein, Mprotease, Mprotein, NTD, RDRP, and Sprotein, the hydrogen bonds paired  within 0.35 nm were estimated during the 100 ns MD simulations. The average number of  hydrogen bonds between remdesivir and CTD, Eprotein, Mprotease, Mprotein, NTD, RDRP,  and Sprotein was found to be 3, 2, 5, 4, 11, 8, and 5, respectively (Figure 9). The maximum  number of hydrogen bonds was found between remdesivir and NTD. RDRP, Sprotein, and  Mprotease also showed extensive hydrogen bonding with remdesivir. Weak interactions  were reported in case of CTD, Eprotein, and Mprotein.  
        </p>
        <p><img src="https://www.frontiersin.org/files/Articles/710778/fphar-12-710778-HTML/image_m/fphar-12-710778-g009.jpg"></p>
        <hr>
        <h4>Conclusion</h4>
        <br>
        <p>The information on the different targets might be beneficial for the development of  potential drugs. </p>
    </div>
</body>
</html>