<!doctype html>
    <head>
        <meta charset="utf-8" >
        <title>Fluvoxamine</title>
        <link href="Cure Decor.css" rel="stylesheet">
    </head>
      <header>
        <div class="navi">
          <p><a href="index.html">Scan to Cure</a></p>
          <ul>
            <li><a href="#">Genomic Characteritics of Corona Virus</a></li>
              <ul>
                <li><a href="Structure.html">Structure of the Covid Molecule</a></li>
              </ul>
            </li>
              
            <li><a href="#">General Info of COVID Pro-Drugs</a>
              <ul>
                <li><a href="Remdesivier Abstract.html">Remdesivir</a></li>
                <li><a href="Fluvoxamine Abstract.html">Fluvoxamine</a></li>
                <li><a href="Molnupiravir Abstract.html">Molnupiravir</a></li>
              </ul>
            </li>
            <li><a href="#">Mechanism of Action</a>
              <ul>
                <li><a href='Remdesivier.html'>Remdesivir</a></li>
                <li><a href='Fluvoxamine.html'>Fluvoxamine</a></li>
                <li><a href='Molnupiravir.html'>Molnupiravir</a></li>
              </ul>
            </li>
            <li><a href='Similarities & Differences.html'>Similarities & Differences</a></li>
            <li><a href='Literature Sources.html'>Literature Source</a></li>
          </ul>
        </div>
    </header>
    <body class="long">
        <div class = "for">
        <h2>Current Knowledge of Corona Virus & Its Mechanism of Action</h2>
        <br><br>
        <p> The current knowledge of CoV molecular biology is based on the studies of the Betacoronavirus genus including the well-characterized animal CoV model, murine hepatitis virus (MHV) and those that have caused the aforementioned lethal zoonotic infections in the 21st century: MERS-CoV and SARSr-CoV strains (Wong et al., 2020). Studies of the latter since their emergence in 2003 have generated a vast body of literature, and this will be the major focus of this review (Chan et al., 2015; Compton et al., 1993; Luk et al., 2019). 
        </p> <br><br>
        <p>Coronaviruses are a family of enveloped RNA viruses that are distributed widely among mammals and birds, causing principally respiratory or enteric diseases but in some cases neurologic illness or hepatitis (Lai and Holmes, 2001). Infections are transmitted mainly via respiratory and fecal‐oral routes. The most distinctive feature of this viral family is genome size: coronaviruses have the largest genomes among all RNA viruses, including those RNA viruses with segmented genomes. oronaviruses are roughly spherical and moderately pleiomorphic (Fig. 1 ). Virions have typically been reported to have average diameters of 80–120 nm, but extreme sizes as small as 50 nm and as large as 200 nm are occasionally given in the older literature (Oshiro 1973, McIntosh 1974). The surface spikes or peplomers of these viruses, variously described as club‐like, pear‐shaped, or petal‐shaped, project some 17–20 nm from the virion surface (McIntosh, 1974), having a thin base that swells to a width of about 10 nm at the distal extremity (Sugiyama and Amano, 1981). For some coronaviruses a second set of projections, 5–10‐nm long, forms an undergrowth beneath the major spikes (Guy 2000, Patel 1982, Sugiyama 1981). These shorter structures are now known to be the hemagglutinin‐esterase (HE) protein that is found in a subset of group 2 coronaviruses (Section III.G). There are three protein components of the viral envelope (Fig. 1). The most prominent of these is the S glycoprotein (formerly called E2) (Cavanagh, 1995), which mediates receptor attachment and viral and host cell membrane fusion (Collins et al., 1982). The S protein is a very large, N‐exo, C‐endo transmembrane protein that assembles into trimers (Delmas 1990, Song 2004) to form the distinctive surface spikes of coronaviruses (Fig. 2 ).</p>
        <p><strong>FIGURE. 1</strong><img src="https://ars.els-cdn.com/content/image/3-s2.0-B9780128012383025502-f02550-02-9780128012383.jpg"></p>
        <br>
        <p><strong>FIGURE. 2</strong><img src="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7150122/bin/gr2.jpg"></p>
        <p>The genomes of coronaviruses are nonsegmented, single‐stranded RNA molecules of positive sense, that is, the same sense as mRNA (Fig. 4 ) (Lai 1978, Lomniczi 1977, Schochetman 1977, Wege 1978). Structurally they resemble most eukaryotic mRNAs, in having both 5′ caps (Lai and Stohlman, 1981) and 3′ poly(A) tails (Lai 1978, Lomniczi 1977, Schochetman 1977, Wege 1978). Unlike most eukaryotic mRNAs, coronavirus genomes are extremely large—nearly three times the size of alphavirus and flavivirus genomes and four times the size of picornavirus genomes.
        </p>
        <p><strong>FIGURE. 3</strong><img src="https://basicmedicalkey.com/wp-content/uploads/2016/08/DA1DB2DC6C28FF4.jpg"></p>
        <p>In common with almost all other positive‐sense RNA viruses, the genomic RNA of coronaviruses is infectious when transfected into permissive host cells. The genome has multiple functions during infection. It acts initially as an mRNA that is translated into the huge replicase polyprotein, the complete synthesis of which requires a ribosomal frameshifting event. The replicase is the only translation product derived from the genome. Coronavirus infections are initiated by the binding of virions to cellular receptors (Fig. 5 ). This sets off a series of events culminating in the deposition of the nucleocapsid into the cytoplasm, where the viral genome becomes available for translation. The positive‐sense genome, which also serves as the first mRNA of infection, is translated into the enormous replicase polyprotein. The replicase then uses the genome as the template for the synthesis, via negative‐strand intermediates, of both progeny genomes and a set of subgenomic mRNAs. At the cell surface S protein can cause the fusion of an infected cell with adjacent, uninfected cells, leading to the formation of large, multinucleate syncytia. This enables the spread of infection independent of the action of extracellular virus, thereby providing some measure of escape from immune surveillance.
        </p>
        <p><strong>FIGURE. 4</strong><img src="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7150122/bin/gr3.jpg"></p>
        <p>The binding of spike to its cellular receptor triggers a major conformational change in the S molecule. In some cases, induction of this conformational change may also require a shift to an acidic pH. It has been realized that the coronavirus S protein is a type I viral fusion protein with functional similarities to the fusion proteins of phylogenetically distant RNA viruses such as influenza virus, HIV, and Ebola virus (Bosch et al., 2003).
            <br><br>
            Although a variety of positive‐ and negative‐strand viral RNA species are synthesized during the course of infection (Section V), coronaviruses selectively incorporate genomic (positive‐strand) RNA into assembled virions. This may be accomplished with varying degrees of stringency by different members of the family. Sucrose gradient‐purified virions of MHV have been found to exclusively contain genomic RNA (Makino et al., 1990). By contrast, similarly purified virions of BCoV (Hofmann et al., 1990), TGEV (Sethna 1989, Sethna 1991), and IBV (Zhao et al., 1993) have been reported to contain significant quantities of subgenomic mRNA, in some cases in molar amounts exceeding those of the genomic RNA. 
            <br><br>
            An important feature of coronavirus RNA synthesis is the high rate of homologous and nonhomologous RNA–RNA recombination that has been demonstrated to occur among selected and unselected markers during the course of infection. Although most experimental work in this area has been performed with MHV (Keck 1987, Keck 1988a, Keck 1988b, Makino 1986, Makino 1987), a high frequency of homologous recombination is clearly an attribute of the entire coronavirus family, given that it has been observed in other viruses in all three groups: TGEV (Sanchez et al., 1999), FIPV (Haijema 2003, Herrewegh 1998), BCV (Chang et al., 1996), and IBV (Cavanagh 1992, Kottier 1995, Kusters 1990, Wang 1993).
            <br><br>
            Most evidence supports a model for viral RNA recombination having three mechanistic requirements (Lai, 1992). First, the RNA polymerase must pause during synthesis. This may be an intrinsic property of the enzyme, or it may result from the enzyme encountering a template secondary structure that exceeds a certain stability threshold. Second, a new template must be in physical proximity. Third, some property of the new template must allow the transfer of the nascent RNA strand and the resumption of RNA synthesis. Alternatively, strand transfer could result from a processive mechanism that does not require polymerase dissociation (Jarvis and Kirkegaard, 1991).
        </p>
      </div>
    </body>
</html>